README.md

```diff
@@ -1,165 +1,261 @@
-- 2.1.2 - Updated documentation and corrected JSON formatting
-- 2.1.1 - Integrated PR 48 and fixed issue with adapt create command
-- 2.1.0 - Fully updated update and install commands
-- 2.0.7 - Added update command, rewritten install command and updated dependencies; targetting Node v8+
-- 2.0.6 - Dependencies updated, registry now targeted consistently, fixed issue #49, CI target set to all Node LTS versions
-- 2.0.5 - Added rename and unregister commands and corrected some typos
-- 2.0.4 - Corrected version numbers
-- 2.0.3 - Fixed issue#53 - npm dependency is git url
-- 2.0.2 - Fixed issue#32 - Error: Cannot find module 'Q' on case sensitive file systems
-- 2.0.1 - Fixed an issue with installing dependencies (https://github.com/adaptlearning/adapt-cli/tree/hotfix/has-no-method-install)
-- 2.0.0 - version bump to 2.0.0
-- 0.0.24 - Pre v2.0.0 release update
-- 0.0.23 - Updated NPM Module's version.
-- 0.0.22 - Added the 'ls' command.
-- 0.0.21 - Fixed an issue where the new slug library does not maintain case.
-- 0.0.19 - Removed slug library due to annoying python dependency, added 'create component' command.
-- 0.0.18 - Fixed Issue #23
-- 0.0.17 - Added extra check current working directory for installing. See Issue #23
-- 0.0.16 - Improved download progress bar; fixed dependency of the download package to a fork to avoid version conflict with decompress package. This should be temporary until the conflict is resolved.
-- 0.0.15 - fixed Issue #22
-- 0.0.14 - fixed Issue #15
-- 0.0.13 - Added support for versions of plugins (#14) and --version command
-- 0.0.12 - fixed Issue #13
-- 0.0.11 - fixed Issue #12
-- 0.0.10 - fixed Issue #2 & #8
-- 0.0.9  - fixed Issue #7
-- 0.0.8  - Added 'create' command, fixed Issue #6
-- 0.0.7  - fixed Issue #3
-- 0.0.6  - Added 'register' command
-- 0.0.5  - Added adapt.json (dependency list)
-- 0.0.3  - Added uninstall command
-...
+The Adapt community maintains the [Adapt Plugin Browser](https://www.adaptlearning.org/index.php/plugin-browser/) as a convenient registry of components, extensions, themes, and menus. The plug-in system is powered by [Bower](http://bower.io/): http://adapt-bower-repository.herokuapp.com/packages/. To register, a plug-in must be a valid bower package with *bower.json*, and have a unique name that is prefixed with "adapt-".  
+
+See [Developing plug-ins](https://github.com/adaptlearning/adapt_framework/wiki/Developing-plugins) for more information on defining your plug-in's package and on [registering your plug-in](https://github.com/adaptlearning/adapt_framework/wiki/Registering-a-plugin).  
+<div float align=right><a href="#top">Back to Top</a></div>  
+
+----------------------------
+**Version number:**  2.1.2   <a href="https://community.adaptlearning.org/" target="_blank"><img src="https://github.com/adaptlearning/documentation/blob/master/04_wiki_assets/plug-ins/images/adapt-logo-mrgn-lft.jpg" alt="adapt learning logo" align="right"></a>   
```

Comment:
Please bump to [latest version](https://github.com/adaptlearning/adapt-cli/blob/master/package.json#L3).

Reply:
done